lib/archethic/db/embedded_impl/chain_reader.ex

```diff
@@ -120,6 +120,15 @@ defmodule Archethic.DB.EmbeddedImpl.ChainReader do
   end
 
   defp do_process_get_chain(fd, address, fields) do
+    # Always return transaction address
+    fields =
+      with false <- Enum.empty?(fields),
```

Comment:
If we want to avoid to check the list few times just to include a field you could leverage either:
- Enum.uniq -> `Enum.uniq([ :address | fields ])`
- MapSet -> `MapSet.new(fields) |> MapSet.union(MapSet.new([:address])`